django/template/base.py

```diff
@@ -393,28 +393,34 @@ def create_token(self, token_string, position, lineno, in_tag):
 
 
 class DebugLexer(Lexer):
+    # This parallels the use of tag_re.split() in Lexer.tokenize().
+    def _tag_re_split(self):
+        prev_end = 0
+        for match in tag_re.finditer(self.template_string):
+            start, end = match.span()
+            yield (prev_end, start)
+            yield (start, end)
+            prev_end = end
+        yield (prev_end, len(self.template_string))
```

Comment:
On the basis of the previous comment, this could be:

```suggestion
    def split():
        string = self.template_string
        last = 0
        for match in tag_re.finditer(string):
            start, end = match.span()
            yield string[last:start], (last, start)
            yield string[start:end], (start, end)
            last = end
        yield string[last:], (last, len(string))
```